content/design-systems-for-developers/react/en/architecture.md

```diff
@@ -87,7 +87,7 @@ Following this method, we end up with UI primitives: Avatar, Badge, Button, Chec
 Add the components we found by downloading them to your computer and dropping them into your repository, as well as removing the application files that Create React App provided:
 
```

Comment:
@DaleSeo the paragraph could probably be a bit improved. We could probably go with :

"At this stage it's safe to delete the `src` folder that Create React App created for us when we initialized our design system earlier. Don't worry we won't be needing it. Then you can add the components already found by downloading them to your computer and also into your local repository:"

```bash 

rm -rf src

svn export https://github.com/chromaui/learnstorybook-design-system/tags/download-1/src
```


@tmeasday and @domyen are you both ok with this small change to the phrasing?

Reply:
LGTM

Reply:
Nice phrasing 👏

Reply:
@DaleSeo i would like to ask you if you still have the time to address this?